modules/ROOT/pages/cassandra/cassandra-design-center.adoc

```diff
@@ -0,0 +1,118 @@
+= Design Center - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configure the Cassandra Connector in Design Center
+. From Anypoint Platform > Design Center, create a Mule Application, and add an HTTP Listener as the trigger.
```

Comment:
I think we are using the term app now and not application, but this may be different for the connector doc. Application should be lowercase.

Reply:
Don't know, can you find out for sure?